test/Utils/TestServices.h

```diff
@@ -24,184 +32,40 @@
 // sortie standard vers le fichier ./results/Integer_full.txt et ensuite
 // comparer ce fichier avec le fichier de reference
 //  ./results.ref/Integer_full.txt. Ce fichier doit avoir ete cree par ailleurs
-//  lors de la mise au point de la
-// methode Integer::Test().
+//  lors de la mise au point de la methode Integer::Test().
 //
-// Le test est reussit lorsque les 2 fichiers sont identiques (a l'exception des
+// Le test est reussi lorsque les 2 fichiers sont identiques (a l'exception des
 // lignes qui commencent par "SYS" qui peuvent etre differentes)
 //
+// clang-format off
```

Comment:
Why clang format is shut down ?

Reply:
I removed it 
In the old big macro clang was very bad but it seems OK now